demisto_sdk/commands/common/errors.py

```diff
@@ -2361,11 +2366,37 @@ def invalid_defaultvalue_for_checkbox_field(name: str):
 
     @staticmethod
     @error_code_decorator
-    def missing_reliability_parameter(command: str):
+    def missing_reliability_parameter(is_feed: bool, command_name: str | None = None):
+        """
+        Returns an error message for missing reliability parameter, according to provided arguments.
+
+        Args:
+            is_feed (bool): Whether the integration is a feed integration or not.
+            command_name (str | None, optional): The name of the command that is missing the reliability parameter.
+                Defaults to None. Used on error message when is_feed is False.
+
+        Returns:
+            str: The error message.
+        """
+        if is_feed:
+            specific_case_error = (
+                "Feed integrations must implement a reliability parameter."
+            )
+
+        else:
+            specific_case_error = (
+                "Integrations with reputation commands{0} ".format(
+                    f" ('{command_name}')"  # Add reputation command's name to error if provided
```

Comment:
I don't think this comment is needed, it is understandable from the code

Reply:
```suggestion
                    f" ('{command_name}')"
```